# Data exploration

* Prepare for filtering
    * Create a directory in your home folder called `filtering/`.
    * Copy the `input.vcf` file to the new directory.
    * Look at the plink documentation and find a way to *convert* the `vcf` file into `bed/bim/bam` files

* Find genotype files
    * How many individuals are in the data?
    * How many SNPs are in the data?


### Paths
VCF path: `/mnt/appbio_a/groups/appbio/G_Louloudis/input.vcf`